### PR TITLE
Add CLI tools for services and CRUD

### PR DESCRIPTION
**Pull request by YuanmingLeee**
Closes #

TODO:
- [x] Start / Stop services
- [x] CRUD management
~- [ ] Converter~
~- [ ] Deployer~

#### Does this change require updates to documentation?
Yes.

#### Before submitting this PR, please make sure you have:

- [x] run `python -m pytest tests/` and confirmed all checks still pass.
- [x] verified that any code or assets from external sources are properly credited.

### CODE REVIEW

File: modelci/hub/cli.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
```
Comment (HuaizhengZhang) [inline modelci/hub/cli.py:1]: all cli related codes should be in the cli directory for easy management.

File: modelci/cli.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Author: Li Yuanming
+Email: yli056@e.ntu.edu.sg
+Date: 10/12/2020
+"""
+import click
+
+import modelci.hub.cli
```
Comment (HuaizhengZhang) [inline modelci/cli.py:10]: from hub import cli is weird
we should have a CLI directory.
we'd better use a hub_cli.py in a CLI directory.

reorgainize this

**Comment by HuaizhengZhang** [comment]
I have changed the PR title.
First, finish this and then you can make a new PR CLI for converter and deployer

File: modelci/cli.py (hunk 2 above)
Comment (YuanmingLeee) [reply modelci/cli.py:10]: This hierarchy is from mlflow. They have sperate cli.py under each module.
For example:
CLI entry point `cli.py`:
https://github.com/mlflow/mlflow/blob/master/mlflow/cli.py
CLI for SageMaker under `sagemaker/cli.py`:
https://github.com/mlflow/mlflow/blob/master/mlflow/sagemaker/cli.py

Comment (YuanmingLeee) [reply modelci/cli.py]: refactored

**Review by HuaizhengZhang — APPROVED** [review]

**Pull request merged**